util/dvsim/dvsim.py

```diff
@@ -84,14 +83,17 @@ def resolve_branch(arg_branch):
 
 # Get the project root directory path - this is used to construct the full paths
 def get_proj_root():
-    result = subprocess.run(["git", "rev-parse", "--show-toplevel"],
-                             stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    cmd = ["git", "rev-parse", "--show-toplevel"]
+    result = subprocess.run(cmd,
+                            stdout=subprocess.PIPE,
+                            stderr=subprocess.PIPE)
     proj_root = result.stdout.decode("utf-8").strip()
-    if proj_root == "":
+    if not proj_root:
         log.error(
-            "Attempted to find the root of this GitHub repository by running:" \
-            "\n..\nBut this command has failed:\n%s" %
-                result.stderr.decode("utf-8"))
+            "Attempted to find the root of this GitHub repository by running:\n"
+            "{}\n"
+            "But this command has failed:\n"
+            "{}".format(' '.join(cmd), result.stderr.decode("utf-8")))
```

Comment:
Minor nit: on lines 54-55 above, you use the `%` syntax to format the string argument of `log.warning()`, and here you use the `.format()` syntax, could you use either one or the other?

Reply:
Ok, I'll change both to use `format`.